mob.go

```diff
@@ -188,12 +234,20 @@ func main() {
 	say.TurnOnDebuggingByArgs(os.Args)
 	say.Debug(runtime.Version())
 
-	if !isGitInstalled() {
+	versionString, currentVersion := gitVersion()
+	if versionString == "" {
 		say.Error("'git' command was not found in PATH. It may be not installed. " +
 			"To learn how to install 'git' refer to https://git-scm.com/book/en/v2/Getting-Started-Installing-Git.")
 		exit(1)
 	}
 
+	if currentVersion.Less(GitVersion{minimumGitVersionMajor, minimumGitVersionMinor, minimumGitVersionPatch}) {
+		say.Error(fmt.Sprintf("'git' command version '%s' is lower than the required minimum version (%d.%d.%d). "+
+			"Please update your 'git' installation!",
+			versionString, minimumGitVersionMajor, minimumGitVersionMinor, minimumGitVersionPatch))
+		exit(1)
```

Comment:
I find this erro very harsh. Could we start with a warning instead?

Reply:
I don’t agree here. If I remember correctly we had issues with basic functionality of mob if you have a older version. The let’s fail fast but then the user definitely sees where the problem is. If we print a warning the user gets another error and maybe find out that his version is to old because there was a warning somewhere at the beginning of the output.

Reply:
I have checked what functionality needs 2.13.0, and it is the `git branch --format=...` calls ([git 2.13 release notes](https://github.blog/2017-05-10-git-2-13-has-been-released/#bits-and-bobs). Currently `gitBranches` and `gitRemoteBranches` uses the `--format` parameter.
For example running mob start with this error turned to a warning will result in this:
```
⚠ 'git' command version 'git version 2.12.5' is lower than the required minimum version (2.13.0). Please update your 'git' installation!
  git fetch origin --prune
ERROR git branch --format=%(refname:short)
ERROR error: unknown option `format=%(refname:short)'
ERROR usage: git branch [<options>] [-r | -a] [--merged | --no-merged]
ERROR or: git branch [<options>] [-l] [-f] <branch-name> [<start-point>]
ERROR or: git branch [<options>] [-r] (-d | -D) <branch-name>...
ERROR or: git branch [<options>] (-m | -M) [<old-branch>] <new-branch>
ERROR or: git branch [<options>] [-r | -a] [--points-at]
ERROR
ERROR Generic options
ERROR -v, --verbose         show hash and subject, give twice for upstream branch
ERROR -q, --quiet           suppress informational messages
ERROR -t, --track           set up tracking mode (see git-pull(1))
ERROR --set-upstream        change upstream info
ERROR -u, --set-upstream-to <upstream>
ERROR change the upstream info
ERROR --unset-upstream      Unset the upstream info
ERROR --color[=<when>]      use colored output
ERROR -r, --remotes         act on remote-tracking branches
ERROR --contains <commit>   print only branches that contain the commit
ERROR --abbrev[=<n>]        use <n> digits to display SHA-1s
ERROR
ERROR Specific git-branch actions:
ERROR -a, --all             list both remote-tracking and local branches
ERROR -d, --delete          delete fully merged branch
ERROR -D                    delete branch (even if not merged)
ERROR -m, --move            move/rename a branch and its reflog
ERROR -M                    move/rename a branch, even if target exists
ERROR --list                list branch names
ERROR -l, --create-reflog   create the branch's reflog
ERROR --edit-description    edit the description for the branch
ERROR -f, --force           force creation, move/rename, deletion
ERROR --merged <commit>     print only branches that are merged
ERROR --no-merged <commit>  print only branches that are not merged
ERROR --column[=<style>]    list branches in columns
ERROR --sort <key>          field name to sort on
ERROR --points-at <object>  print only branches of the object
ERROR -i, --ignore-case     sorting and filtering are case insensitive
ERROR exit status 129
```

I think this is not user-friendly error message, while the user will have the warning at the top of the error, more than 90% of the console output is filled with the git output. I would have to agree with @hollesse here about displaying an error and stopping. Although we can reword the error message if you find the wording too harsh.